README.md

```diff
@@ -72,3 +72,29 @@ pip install pre-commit
 Then run: `pre-commit install`
 
 And to install golangci-lint please follow the instructions [here](https://golangci-lint.run/welcome/install/#local-installation).
+
+# Integration Tests
+
+## Run fake-gcs-server
+
+Integration tests require disk space and [fake-gcs-server](https://github.com/fsouza/fake-gcs-server) to be running on your machine.
+
+To run integration tests, you can start `fake-gcs-server` in Docker with the following command:
+
+```bash
+docker run -d --name fake-gcs-server -p 4443:4443 fsouza/fake-gcs-server -scheme http -public-host 0.0.0.0:4443
+```
+
+Notes:
+
+The `-scheme http` and `-public-host 0.0.0.0:4443` flags are required to run the tests correctly:
+- `-public-host 0.0.0.0:4443` ensures that readers are created correctly. For more details, refer to [this issue](https://github.com/fsouza/fake-gcs-server/issues/201).
+- `-scheme http` is required for proper local hosting without TLS checks. The GCS client has issues connecting to HTTPS servers, as it verifies the TLS certificate and fails in this setup.
+
+## Run Integration Tests
+
+Use the following command to run the integration tests:
+
+bash```
+STORAGE_EMULATOR_HOST=http://0.0.0.0:4443 go test ./...
```

Comment:
make test?

Reply:
@amanjpro suggested to add

Reply:
I think what he says is in the README, we no longer this detail, because we already have the Make target, and I agree

We probably just need to tell you cannot run the tests under a windows machine